src/styles/index.scss

```diff
@@ -64,15 +64,25 @@ input {
   color: var(--text-primary);
 }
 
-h1,
+h1 {
```

Comment:
**question**: changes to these header styles will affect the entire app. Are we sure we want this?

It's not great, but we could instead apply inline styles where we need to.

Reply:
Yep agreed, I've just added these as inline styles @JayZeeDesign